examples/notebooks/create_deployment.ipynb

```diff
@@ -12,48 +23,288 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load config from default location."
+   ]
+  },
+  {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {
     "collapsed": true
    },
    "outputs": [],
    "source": [
-    "client.Configuration().host=\"http://localhost:8080\""
+    "config.load_kube_config()\n",
+    "extensions = client.ExtensionsV1beta1Api()"
```

Comment:
I also changed this line in:

https://github.com/kubernetes-incubator/client-python/pull/242/files

I think it's better to load incluster config as the examples are intended to be run from inside a kubernetes cluster.